code.py

```diff
@@ -106,7 +106,7 @@ def showImage(imageFile):
 # Set up wifi connection
 spi = busio.SPI(board.SCK, board.MOSI, board.MISO)
 connection = Connection()
-wifi_manager = connection.connect(spi, True) 

```

Comment:
for my understanding, can you give a quick explanation of how the new miniMQTT version no longer makes use of the wifi_manager?

Reply:
It was changed to have a similar interface to the requests module - so you set the socket at the global level rather than pass the connection in when creating the MQTT instance.

If you look in connection.py you will see the socket set after the requests socket is set.